<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrAnaViFocuse</class>
 <widget class="QWidget" name="IrAnaViFocuse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IrAnaViFocuse</string>
  </property>
  <widget class="QPushButton" name="id_btn_circle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_delete_shape">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_vi_focus_near">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>可见光焦点近</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_rect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>框</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_vi_focus_far">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>可见光焦点远</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_ir_focus_far">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>红外焦点远</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_ir_focus_near">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>红外焦点近</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_point">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>点</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
